src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/ErrorMsg.java

```diff
@@ -27,7 +28,8 @@ public ErrorMsg(
         @JsonProperty("poBox") String poBox,
         @JsonProperty("documentControlNumber") String documentControlNumber,
         @JsonProperty(value = "errorCode", required = true) ErrorCode errorCode,
-        @JsonProperty(value = "errorDescription", required = true) String errorDescription
+        @JsonProperty(value = "errorDescription", required = true) String errorDescription,
+        @JsonProperty(value = "service") String service
```

Comment:
Do not need 'value' here

Reply:
I'm leaving it as it is. Because service will be made mandatory in the next pr